net-tools/netwatch/src/interfaces.rs

```diff
@@ -89,7 +89,6 @@ impl Interface {
     /// free to add parameters or different alternative constructors.
     ///
     /// [`ProbePlan`]: crate::netcheck::reportgen::probes::ProbePlan
-    #[cfg(test)]
```

Comment:
why remove this?

Reply:
because the test that uses this is in what is now another crate

Reply:
but it is still `pub(crate)`

Reply:
oh, I see but the caller is not `cfg(test)` anymore, got it